tox.ini

```diff
@@ -3,9 +3,16 @@ envlist = py{27, 36}
 
 [testenv]
 passenv = test_flags
-commands = python setup.py nosetests --xunit-file=nosetests_{envname}.xml {env:test_flags}
+install_command = pip install \
+                        --index-url=http://pypi.camlab.kat.ac.za/pypi/trusty \
```

Comment:
Why is Trusty used here? What goes wrong if it isn't?

Reply:
> Why is Trusty used here? 

- why index-url: I wanted pip to use our local PyPi repository as some of our packages are private and not hosted on pypi.org.
- why trusty: Our current PyPi local server/repository points to trusty (Not sure why, I suppose the idea was to distinguish from trusty and bionic. @ajoubertza would know).
There's also a [JIRA ticket](https://skaafrica.atlassian.net/browse/MT-661) that aims to upgrade our local repo.

> What goes wrong if it isn't?

The initial idea was to standardise/have a generic tox.ini across all cam repos (for easier maintainability). As for katportalclient, this is unnecessary as it does not depend on any of CAM's private packages.

Another motivation was the deprecation of tox's `indexserver`. 
See: https://tox.readthedocs.io/en/latest/config.html#conf-indexserver

Reply:
> * why trusty:

Yes, the plan long was probably to distinguish different platforms, but we're still using this same URL for bionic. For source installs it wouldn't matter.  Maybe the plan was to allow for wheels built on for different platforms.

Can we leave this specific index out of this tox.ini file?  I think that will be better since this repo is public - people cloning it will not have access to our PyPI repository.